style/swift/README.md

```diff
@@ -14,3 +14,5 @@ Swift
 * Group methods into specific extensions for each level of access control
 * When capitalizing acronyms or initialisms, follow the capitalization of the
   first letter.
+* Break up long initializers or function calls by adding linebreaks between
```

Comment:
Good point. I'm planning on working on this document a bit tomorrow for investment time. I'll make a pass at the guidelines to fix the wording then.